UPGRADE.md

```diff
@@ -12,8 +12,9 @@ does not have any particular instructions.
 
 #### Sorting
 
-Policy merging now gives precedence to policies lexicographically before
-other policies.
+This change relates only to the new targetRef policies. When 2 policies have a tie on the targetRef kind we compare their names lexicographically.
+Policy merging now gives precedence to policies that lexicographically "less" than other policies, i.e. policy "aaa" takes precedence over "bbb" because "aaa" < "bbb".
+Previously, before 2.6.0 the order was the opposite.
```

Comment:
Maybe worth mentioning that this fix is introduced to follow what was documented and in the MADR.

Reply:
The initial MADR had the opposite order (it was implemented correctly). We decided to change it in 2.6.0 to be less confusing to users